docs/manuals/source/TasksAndConcepts/Plugins/FileDaemonPlugins.rst.inc

```diff
@@ -24,3 +24,4 @@ File Daemon plugins are configured by the :strong:`Plugin` directive of a :ref:`
 .. include:: Plugins/FileDaemonPlugins/PythonFdPlugin.rst.inc
 .. include:: Plugins/FileDaemonPlugins/VMwarePlugin.rst.inc
 .. include:: Plugins/FileDaemonPlugins/GrpcPlugin.rst.inc
+.. include:: Plugins/FileDaemonPlugins/YuzuyQumuloFdPlugin.rst.inc
```

Comment:
```suggestion
.. include:: Plugins/FileDaemonPlugins/PythonFdPlugin.rst.inc
.. include:: Plugins/FileDaemonPlugins/YuzuyQumuloFdPlugin.rst.inc
.. include:: Plugins/FileDaemonPlugins/VMwarePlugin.rst.inc
.. include:: Plugins/FileDaemonPlugins/GrpcPlugin.rst.inc
```

The section had been intended to be in alphabetic order. Maybe also the Grpc Plugin should be moved, but that is something special and can stay.

Also a would prefer a file name like: QumuloFdPlugin.rst.inc